Source/Scene/Model.js

```diff
@@ -512,11 +512,15 @@ define([
         this.colorBlendAmount = defaultValue(options.colorBlendAmount, 0.5);
 
         /**
-         * The {@link ClippingPlaneCollection} used to selectively disable rendering the model. Clipping planes are not currently supported in Internet Explorer.
+         * The {ClippingPlaneCollection} used to selectively disable rendering the model. Clipping planes are not currently supported in Internet Explorer.
```

Comment:
Remove any occurrences of `Clipping planes are not currently supported in Internet Explorer.`